src/Symfony/Component/Translation/Bridge/Lokalise/LokaliseProvider.php

```diff
@@ -149,6 +156,7 @@ private function exportFiles(array $locales, array $domains): array
                 'filter_langs' => array_values($locales),
                 'filter_filenames' => array_map([$this, 'getLokaliseFilenameFromDomain'], $domains),
                 'export_empty_as' => 'skip',
+                'placeholder_format' => $this->intlIcuEnabled ? 'icu' : 'symfony',
```

Comment:
```suggestion
                'placeholder_format' => $this->intlIcuEnabled ? 'icu' : 'symfony',
                'plural_format' => $this->intlIcuEnabled ? 'icu' : 'symfony',
```

A different parameter is required to ensure that the plurals are also in the expected ICU format. The param is called `plural_format` which should contain the same value as  placeholder_format. See https://developers.lokalise.com/reference/download-files for all params. It would be great if that can be included. Without this option users might get invalid ICU translation files

Reply:
Without this change, you will still get plurals in the old format from Lokalise, which the Symfony Translator is not expecting in ICU files

Reply:
This would be really weird behaviour IMHO, but better safe then sorry i guess :D

Could you verify this?

Reply:
It's weird, but I guess @TwanVermeulen is right. According to the documentation https://developers.lokalise.com/reference/api-plurals-and-placeholders#plural-format-descriptions I guess we have to pass both parameters.

Reply:
I can add it, but as i said - cant really verify for now as I'm not working anymore on the project which needed this patch. So is this verified that this is needed and working?